sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/CloudServiceExtensionProperties.cs

```diff
@@ -132,14 +132,14 @@ public CloudServiceExtensionProperties(string publisher = default(string), strin
         /// Extension (like RDP), this is the XML setting for the extension.
         /// </summary>
         [JsonProperty(PropertyName = "settings")]
-        public string Settings { get; set; }
+        public object Settings { get; set; }
```

Comment:
This seems a breaking change. But looks like there is no customization to fix the backward compatibility.